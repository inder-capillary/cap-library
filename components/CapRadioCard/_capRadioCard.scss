@import "../styles/_variables.scss";

$capRadioCardV2: cap-radioCard-v2;
.ant-radio-group.#{$capRadioCardV2} {
  width: 800px;
  .ant-radio-button-wrapper {
    border-radius: 4px;
    width: 372px;
    height: 120px;
    background-color: $CAP_WHITE;
    margin: 6px;
    padding: 0;
    border-top-width: 1px;
  }
  .ant-card-body {
    display: flex;
  }
  .ant-card.ant-card-bordered {
    height: 100%;
    border: none;
  }
  .radio-card-content-container {
    width: 100%;
  }
  .radio-card-content,
  .radio-card-header {
    height: 47px;
    font-size: $FONT_SIZE_S;
    line-height: 1.17;
    letter-spacing: normal;
    color: $CAP_G04;
  }

  .radio-card-header {
    height: 25px;
  }

  .radio-card-icon {
    width: 48px;
    height: 48px;
    object-fit: contain;
    margin-right: $CAP_SPACE_24;
  }

  .radio-card-checked {
    position: absolute;
    right: -6px;
    top: -12px;
    display: none;
  }

  .ant-radio-button-wrapper-checked {
    border-radius: 4px;
    .radio-card-checked {
      display: block;
      z-index: 1;
    }
  }
  .ant-radio-button-wrapper-disabled {
    opacity: 0.5;
  }
  .div-icon {
    position: absolute;
    top: 25%;
    left: 25%;
  }
  .icon-container {
    padding-right: 24px;
    height: 48px;
    width: 48px;
    background-color: $CAP_G07;
    background-size: cover;
    text-decoration: none;
    display: inline-block;
    border-radius: 18px;
    position: relative;
    &.green-color {
      background-color: map-get($CAP_PRIMARY, base);
      .cap-icon-v2 {
        color: $CAP_WHITE;
      }
    }
  }
}
