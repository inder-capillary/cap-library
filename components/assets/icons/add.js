import React from 'react';

const addSvg = (props) => (
    <svg width="16" height="16" viewBox="0 0 16 16" {...props}>
        <defs>
            <path id="a" d="M9 7h4a1 1 0 0 1 0 2H9v4a1 1 0 0 1-2 0V9H3a1 1 0 1 1 0-2h4V3a1 1 0 1 1 2 0v4z" />
        </defs>
        <g fill="none" fillRule="evenodd">
            <mask id="b" fill="#fff">
                <use xlinkHref="#a" />
            </mask>
            <use fill="#006CC7" xlinkHref="#a" />
            <g fill={props.color} mask="url(#b)">
                <path d="M0 0h16v16H0z" />
            </g>
        </g>
    </svg>
)

addSvg.defaultProps = {
    color: "#2466EA"
}

export default addSvg
