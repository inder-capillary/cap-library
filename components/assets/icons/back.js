const SvgBack = props => (
  <svg width={24} height={24} {...props}>
    <defs>
      <path
        d="M20 12a1 1 0 0 1-1 1H8l4.79 4.79a1.004 1.004 0 1 1-1.42 1.42l-6.503-6.503a1 1 0 0 1 0-1.414L11.37 4.79a1.004 1.004 0 1 1 1.42 1.42L8 11h11a1 1 0 0 1 1 1z"
        id="back_svg__a"
      />
    </defs>
    <g fill="none" fillRule="evenodd">
      <mask id="back_svg__b" fill="#fff">
        <use xlinkHref="#back_svg__a" />
      </mask>
      <use fill="#FFF" xlinkHref="#back_svg__a" />
      <g mask="url(#back_svg__b)" fill="#091E42">
        <path d="M0 0h24v24H0z" />
      </g>
    </g>
  </svg>
);

export default SvgBack;