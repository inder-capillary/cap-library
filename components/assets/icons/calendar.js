import React from 'react';

const Calender = (props) => (
    <svg width="24" height="24" viewBox="0 0 24 24" {...props}>
        <defs>
            <path id="a" d="M5 7v12h14V7H5zm13-2h1a2 2 0 0 1 2 2v12a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V7a2 2 0 0 1 2-2h1V4a1 1 0 1 1 2 0v1h8V4a1 1 0 0 1 2 0v1zm1 4H5v2h14V9zm-5 5v3h3v-3h-3z" />
        </defs>
        <g fill="none" fillRule="evenodd">
            <mask id="b" fill="#fff">
                <use xlinkHref="#a" />
            </mask>
            <use fill="#7A869A" xlinkHref="#a" />
            <g fill="#091E42" mask="url(#b)">
                <path d="M0 0h24v24H0z" />
            </g>
        </g>
    </svg>
)

export default Calender;

