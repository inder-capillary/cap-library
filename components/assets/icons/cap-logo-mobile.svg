<?xml version="1.0" encoding="UTF-8"?>
<svg width="28px" height="28px" viewBox="0 0 28 28" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <title>Group 3</title>
    <defs>
        <linearGradient x1="58.9508296%" y1="-24.7474463%" x2="32.9109837%" y2="106.740605%" id="linearGradient-1">
            <stop stop-color="#F05A24" offset="0%"></stop>
            <stop stop-color="#F05D24" offset="20%"></stop>
            <stop stop-color="#F58E16" offset="32%"></stop>
            <stop stop-color="#FDDB00" offset="52%"></stop>
            <stop stop-color="#EAD707" offset="55%"></stop>
            <stop stop-color="#AACA1F" offset="65%"></stop>
            <stop stop-color="#78C031" offset="75%"></stop>
            <stop stop-color="#54B83F" offset="83%"></stop>
            <stop stop-color="#3EB447" offset="90%"></stop>
            <stop stop-color="#36B24A" offset="94%"></stop>
        </linearGradient>
        <linearGradient x1="17.8919182%" y1="74.1623826%" x2="79.4303797%" y2="41.6183298%" id="linearGradient-2">
            <stop stop-color="#CF4F27" offset="0%"></stop>
            <stop stop-color="#D95226" stop-opacity="0.69" offset="32%"></stop>
            <stop stop-color="#F05A24" stop-opacity="0" offset="100%"></stop>
        </linearGradient>
        <linearGradient x1="42.3856856%" y1="13.6472946%" x2="48.0513492%" y2="44.9098196%" id="linearGradient-3">
            <stop stop-color="#76933C" offset="0%"></stop>
            <stop stop-color="#2E9C48" stop-opacity="0" offset="100%"></stop>
        </linearGradient>
        <linearGradient x1="27.1632448%" y1="26.4959736%" x2="59.2223154%" y2="67.464422%" id="linearGradient-4">
            <stop stop-color="#1687C8" offset="0%"></stop>
            <stop stop-color="#1687C8" offset="29%"></stop>
            <stop stop-color="#1688C7" offset="30%"></stop>
            <stop stop-color="#22979A" offset="46%"></stop>
            <stop stop-color="#2AA377" offset="61%"></stop>
            <stop stop-color="#31AB5E" offset="76%"></stop>
            <stop stop-color="#35B04F" offset="89%"></stop>
            <stop stop-color="#36B24A" offset="100%"></stop>
        </linearGradient>
        <linearGradient x1="39.7283334%" y1="32.15311%" x2="79.9764963%" y2="101.881978%" id="linearGradient-5">
            <stop stop-color="#36B24A" stop-opacity="0" offset="0%"></stop>
            <stop stop-color="#2DAF4A" stop-opacity="0.21" offset="14%"></stop>
            <stop stop-color="#1AA84A" stop-opacity="0.63" offset="46%"></stop>
            <stop stop-color="#0EA44A" stop-opacity="0.9" offset="68%"></stop>
            <stop stop-color="#09A24A" offset="80%"></stop>
        </linearGradient>
        <linearGradient x1="125.928511%" y1="89.5596384%" x2="2.05628642%" y2="10.4695246%" id="linearGradient-6">
            <stop stop-color="#0EAB4B" offset="26%"></stop>
            <stop stop-color="#10A366" stop-opacity="0.78" offset="39%"></stop>
            <stop stop-color="#148FAB" stop-opacity="0.23" offset="67%"></stop>
            <stop stop-color="#1687C8" stop-opacity="0" offset="78%"></stop>
        </linearGradient>
        <linearGradient x1="125.928511%" y1="89.5596384%" x2="2.05628642%" y2="10.4695246%" id="linearGradient-7">
            <stop stop-color="#038A45" offset="26%"></stop>
            <stop stop-color="#088968" stop-opacity="0.73" offset="41%"></stop>
            <stop stop-color="#1587C2" stop-opacity="0.05" offset="76%"></stop>
            <stop stop-color="#1687C8" stop-opacity="0" offset="78%"></stop>
        </linearGradient>
        <linearGradient x1="66.2254233%" y1="107.059335%" x2="46.4145545%" y2="26.5204965%" id="linearGradient-8">
            <stop stop-color="#FDDB00" offset="0%"></stop>
            <stop stop-color="#FDD901" offset="34%"></stop>
            <stop stop-color="#FCD104" offset="52%"></stop>
            <stop stop-color="#FAC308" offset="67%"></stop>
            <stop stop-color="#F8B00F" offset="79%"></stop>
            <stop stop-color="#F69818" offset="90%"></stop>
            <stop stop-color="#F38020" offset="98%"></stop>
        </linearGradient>
        <linearGradient x1="76.2960548%" y1="18.1656727%" x2="38.3436583%" y2="63.0820277%" id="linearGradient-9">
            <stop stop-color="#CF4F27" offset="0%"></stop>
            <stop stop-color="#D75226" stop-opacity="0.75" offset="18%"></stop>
            <stop stop-color="#E05525" stop-opacity="0.48" offset="38%"></stop>
            <stop stop-color="#E75725" stop-opacity="0.27" offset="58%"></stop>
            <stop stop-color="#EC5924" stop-opacity="0.12" offset="75%"></stop>
            <stop stop-color="#EF5A24" stop-opacity="0.03" offset="90%"></stop>
            <stop stop-color="#F05A24" stop-opacity="0" offset="100%"></stop>
        </linearGradient>
        <linearGradient x1="4.5587066%" y1="29.2375858%" x2="72.7272727%" y2="78.854646%" id="linearGradient-10">
            <stop stop-color="#67BB46" offset="0%"></stop>
            <stop stop-color="#A9D033" offset="38%"></stop>
            <stop stop-color="#CBDB2A" offset="56%"></stop>
            <stop stop-color="#D8D91F" offset="71%"></stop>
            <stop stop-color="#FAD502" offset="98%"></stop>
            <stop stop-color="#FDD500" offset="100%"></stop>
        </linearGradient>
        <linearGradient x1="4.5587066%" y1="29.2375858%" x2="72.7272727%" y2="78.854646%" id="linearGradient-11">
            <stop stop-color="#029347" offset="0%"></stop>
            <stop stop-color="#3DA83E" stop-opacity="0.7" offset="17%"></stop>
            <stop stop-color="#70BA37" stop-opacity="0.45" offset="34%"></stop>
            <stop stop-color="#98C931" stop-opacity="0.25" offset="51%"></stop>
            <stop stop-color="#B4D32D" stop-opacity="0.11" offset="67%"></stop>
            <stop stop-color="#C5D92B" stop-opacity="0.03" offset="84%"></stop>
            <stop stop-color="#CBDB2A" stop-opacity="0" offset="100%"></stop>
        </linearGradient>
    </defs>
    <g id="Mobile_Reports-IA-and-listing" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="01.01.00_Mobile_Report-IA_Org-selection" transform="translate(-16.000000, -24.000000)">
            <g id="Group-3" transform="translate(16.000000, 24.000000)">
                <g id="product-logo-copy"></g>
                <g id="Group-2" transform="translate(2.000000, 5.000000)">
                    <path d="M11.9228408,2.29821382 C12.3357441,2.58724608 12.8776795,2.97047188 13.3176795,3.31627833 C19.0428408,0.308536398 22.6725182,5.29305253 19.1241311,8.49434285 C18.6643515,8.89440135 18.1604511,9.24072462 17.6221957,9.52660091 C15.8157441,10.5498267 13.1370344,11.958859 12.5254215,14.1717622 C12.2054215,15.3330525 12.5602602,16.7072461 13.5254215,17.434988 C14.2583247,17.9885364 15.2028408,18.1059558 16.0699376,17.9046654 C19.3615505,17.2143428 20.2712279,13.7020848 19.2880021,10.951117 L19.2609053,10.9678912 C19.7380663,10.667516 20.196149,10.3378516 20.6325182,9.98079446 C28.4712279,3.51111704 20.9267118,-3.76114102 11.9228408,2.29821382 Z M17.547357,11.951117 C17.8647763,15.1769235 14.5280021,16.9833751 13.2557441,16.0220848 C11.983486,15.0607945 15.1344537,13.2453106 16.7860666,12.3601493 L17.547357,11.951117 Z" id="Shape" fill="url(#linearGradient-1)"></path>
                    <path d="M17.223486,2.4169235 L17.223486,0.12918156 C15.6028408,0.362729947 13.7938086,1.04531059 11.9228408,2.30466543 C12.3357441,2.59369769 12.8776795,2.9769235 13.3176795,3.32272995 C14.7860666,2.54466543 16.1150989,2.30079446 17.223486,2.4169235 Z" id="Path" fill="url(#linearGradient-2)"></path>
                    <path d="M19.2918731,10.9472461 L17.547357,11.951117 C17.5974163,12.4386584 17.5593918,12.9312286 17.4350989,13.4053106 L17.4350989,17.3859558 C19.5770344,16.1485364 20.1499376,13.2904719 19.2918731,10.9472461 Z" id="Path" fill="url(#linearGradient-3)"></path>
                    <path d="M13.547357,3.49434285 C13.0209053,3.0801493 12.5150989,2.64272995 11.9602602,2.26724608 C2.50477631,-4.15339909 -5.40490111,4.43498801 4.73445373,11.0130525 C4.98390777,10.3450359 5.2859292,9.69784706 5.63767953,9.07756866 C0.72026018,6.02337511 4.57574405,0.0956331726 10.6105828,3.31627833 C11.0338086,3.51756866 15.7718731,6.33950414 16.9783247,9.8969235 C17.4699376,9.61950414 17.979615,9.35240737 18.4699376,9.01563317 C17.5344537,7.3201493 15.5615505,5.13305253 13.547357,3.49434285 Z" id="Path" fill="url(#linearGradient-4)"></path>
                    <path d="M16.0570344,5.84531059 C15.9912279,5.91369769 15.9280021,5.98079446 15.8557441,6.04272995 C15.7615505,6.12272995 15.663486,6.19885898 15.5628408,6.27111704 C15.4621957,6.34337511 15.3576795,6.41434285 15.2518731,6.48272995 L14.9409053,6.68789124 L14.9254215,6.69821382 C15.8054215,7.63885898 16.583486,8.72402027 16.9783247,9.89047188 C17.4699376,9.61305253 17.979615,9.34595575 18.4699376,9.00918156 C17.9344537,8.0517622 17.0738086,6.93305253 16.0570344,5.84531059 Z" id="Path" fill="url(#linearGradient-5)"></path>
                    <path d="M3.53574405,6.5885364 L3.53574405,6.5885364 C3.34109301,6.76038574 3.13216675,6.91535378 2.91122792,7.0517622 C2.68929244,7.19240737 2.45961502,7.32143962 2.23122792,7.45305253 C2.11767953,7.51756866 2.00413115,7.58208479 1.89187308,7.6517622 C1.83638921,7.68531059 1.78219566,7.7201493 1.72542147,7.75240737 C1.66864728,7.78466543 1.61187308,7.81950414 1.55638921,7.85563317 C1.50090534,7.8917622 1.4183247,7.95369769 1.35380857,8.0001493 C2.11638921,9.0117622 3.22477631,10.0336977 4.73058276,11.0130525 C4.9800368,10.3450359 5.28205824,9.69784706 5.63380857,9.07756866 C4.47509889,8.35627833 3.80542147,7.47369769 3.53574405,6.5885364 Z" id="Path" fill="url(#linearGradient-6)"></path>
                    <path d="M3.53574405,6.5885364 L3.53574405,6.5885364 C3.34109301,6.76038574 3.13216675,6.91535378 2.91122792,7.0517622 C2.68929244,7.19240737 2.45961502,7.32143962 2.23122792,7.45305253 C2.11767953,7.51756866 2.00413115,7.58208479 1.89187308,7.6517622 C1.83638921,7.68531059 1.78219566,7.7201493 1.72542147,7.75240737 C1.66864728,7.78466543 1.61187308,7.81950414 1.55638921,7.85563317 C1.50090534,7.8917622 1.4183247,7.95369769 1.35380857,8.0001493 C2.11638921,9.0117622 3.22477631,10.0336977 4.73058276,11.0130525 C4.9800368,10.3450359 5.28205824,9.69784706 5.63380857,9.07756866 C4.47509889,8.35627833 3.80542147,7.47369769 3.53574405,6.5885364 Z" id="Path" fill="url(#linearGradient-7)"></path>
                    <path d="M7.15380857,9.93563317 C7.19122792,9.85434285 7.2299376,9.77434285 7.26735695,9.69821382 C8.19122792,7.78337511 10.1447763,5.63240737 12.0067118,4.20143962 C11.9163892,4.1369235 11.7305828,4.00402027 11.6428408,3.94337511 C11.3244118,3.71799484 10.9902202,3.51575505 10.6428408,3.33821382 C9.31509889,4.42208479 7.8699376,5.87627833 6.95638921,7.0685364 C5.73961502,8.65821382 4.24671179,10.6814396 4.24671179,13.3085364 C4.24671179,15.4246654 5.3499376,17.3756332 7.85961502,17.9007945 C8.73187308,18.1020848 9.75380857,17.9846654 10.4867118,17.431117 C11.4505828,16.7033751 11.8299376,15.3175687 11.4880021,14.1678912 C10.8028408,11.8917622 7.15380857,9.93563317 7.15380857,9.93563317 Z M10.7744537,15.9343428 C9.72800212,17.0324074 6.58735695,15.5007945 6.39767953,12.8169235 C6.3766374,12.5421122 6.39840214,12.2657 6.46219566,11.9975687 C6.99122792,12.2582138 11.9602602,14.6762783 10.7744537,15.9291816 L10.7744537,15.9343428 Z" id="Shape" fill="url(#linearGradient-8)"></path>
                    <path d="M8.19380857,5.90208479 C8.35810964,6.02939661 8.52413115,6.1545579 8.69187308,6.27756866 C8.85961502,6.40057941 9.02950749,6.51928909 9.2015505,6.63369769 C9.24284083,6.66208479 9.28671179,6.68789124 9.33058276,6.71498801 C10.141466,5.79315989 11.0374758,4.9498566 12.0067118,4.19627833 C11.9163892,4.1317622 11.7305828,3.99885898 11.6428408,3.93821382 C11.3244118,3.71283355 10.9902202,3.51059376 10.6428408,3.33305253 C9.72475888,4.09166227 8.85831823,4.91068577 8.04929244,5.78466543 C8.09703437,5.82466543 8.14477631,5.86466543 8.19380857,5.90208479 Z" id="Path" fill="url(#linearGradient-9)"></path>
                    <path d="M7.15509889,9.93563317 C6.85843967,10.5999847 6.62640242,11.2913434 6.46219566,12.0001493 C6.74090534,12.1291816 8.30606663,12.9033751 9.5099376,13.7666009 L11.3305828,13.7666009 C10.3809053,11.6646654 7.15509889,9.93563317 7.15509889,9.93563317 Z" id="Path" fill="url(#linearGradient-10)"></path>
                    <path d="M7.15509889,9.93563317 C6.85843967,10.5999847 6.62640242,11.2913434 6.46219566,12.0001493 C6.74090534,12.1291816 8.30606663,12.9033751 9.5099376,13.7666009 L11.3305828,13.7666009 C10.3809053,11.6646654 7.15509889,9.93563317 7.15509889,9.93563317 Z" id="Path" fill="url(#linearGradient-11)"></path>
                </g>
            </g>
        </g>
    </g>
</svg>