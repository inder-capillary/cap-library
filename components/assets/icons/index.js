export { default as View } from './view';
export { default as Message } from './message';
export { default as DropDownIcon } from './dropdownIcon';
export { default as SearchIcon } from './searchIcon';
export { default as CloseIcon } from './closeIcon';
export { default as AddIcon } from './add'
export { default as LogoBackground } from './logoBackground';
export { default as Calender } from './calendar';
export { default as SelectedTick } from './selectedTick'
export {default as backIcon} from './back';
export {default as SvgMore} from './more';
export {default as CheckFilled} from './check-filled';
export {default as UserIcon} from './user';
export {default as SurveyIcon} from './survey';
export {default as DynamicIcon} from './dynamic';
export {default as OutboundIcon} from './outbound';
export { default as InfoIcon } from './info';
