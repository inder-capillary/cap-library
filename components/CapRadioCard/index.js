/**
*
* CapRadioCard
*
*/

import React from 'react';
import './_capRadioCard.scss';

import { Card, Radio } from "antd";
import PropTypes from 'prop-types';
import CapColumn from '../CapColumn';
import CapRow from '../CapRow';
import CapHeading from '../CapHeading';
import {CheckFilled } from '../assets/icons/index';
const classNames = require('classnames');

const RadioButton = Radio.Button;
const RadioGroup = Radio.Group;
class CapRadioCard extends React.Component { // eslint-disable-line react/prefer-stateless-function
  render() {
    const {panes, className, onChange, selected} = this.props;
    return (
      <RadioGroup onChange={onChange} className={classNames("cap-radioCard-v2", className)}>
        {panes ? (
          panes.map((pane, i) => {
            const {content, title, icon} = pane;
            return (
              <RadioButton key={i} value={i}>
                <CheckFilled className="radio-card-checked" />
                <Card key={i}>
                  <CapRow key={`capRow_1_${i}`}>
                    {icon ? (
                      <CapColumn className="radio-card-icon" key={`capCol_1_${i}`} span={6}>
                        <div className={(selected === i ? "icons green-color" : "icons")}>
                          <a><div className="div-icon">{icon}</div></a>
                        </div>
                      </CapColumn>
                    ) : <div></div>}
                    <CapColumn key={i} span={18}>
                      <CapRow key={`capRow_2_${i}`} className="radio-card-header"><CapHeading key={i} type="h4">{title}</CapHeading></CapRow>
                      <CapRow key={`capRow_3_${i}`} className="radio-card-content">
                        {' '}
                        {content}
                      </CapRow>
                    </CapColumn>

                  </CapRow>
                </Card>
              </RadioButton>

            );
          }) ) : <div></div>}
      </RadioGroup>

    );
  }
}

CapRadioCard.propTypes = {
  panes: PropTypes.arrayOf(Object),
  className: PropTypes.string,
  onChange: PropTypes.func,
  selected: PropTypes.any,
};

export default CapRadioCard;
