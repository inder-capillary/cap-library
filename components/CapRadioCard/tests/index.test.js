
import renderer from 'react-test-renderer';
import React from 'react'; // eslint-disable-line no-unused-vars
import { Message, View } from "../../assets/icons/index";
import CapRadioCard from '../index';
const panes = [{title: "Outbound", content: "Sending an outbound message is like blasting the message or broadcasting the message to a pre-defined set of users", icon: <Message />},
  {title: "Dynamic", content: "for sending messages to users based on their actions in real time"}, {title: "Survey", content: "for taking survey or feedback from the audience"}, {title: "Referal", content: "for sending messages to acquire new users through existing customers", icon: <View />}];

describe('<CapRadioCard />', () => {
  it('CapRadioCard tenders correctly', () => {
    const tree = renderer.create(<CapRadioCard panes={panes} selected="none" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
