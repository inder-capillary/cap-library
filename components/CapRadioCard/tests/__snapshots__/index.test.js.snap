// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CapRadioCard /> CapRadioCard tenders correctly 1`] = `
<div
  className="ant-radio-group ant-radio-group-outline cap-radioCard-v2"
>
  <label
    className="ant-radio-button-wrapper"
  >
    <span
      className="ant-radio-button"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-button-input"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value={0}
      />
      <span
        className="ant-radio-button-inner"
      />
    </span>
    <span>
      <svg
        className="radio-card-checked"
        height={24}
        viewBox="0 0 48 48"
        width={24}
      >
        <path
          d="M44 24c0 11.045-8.955 20-20 20S4 35.045 4 24 12.955 4 24 4s20 8.955 20 20z"
          fill="#4caf50"
        />
        <path
          d="M34.602 14.602L21 28.199l-5.602-5.598-2.797 2.797L21 33.801l16.398-16.402-2.796-2.797z"
          fill="#ccff90"
        />
      </svg>
      <div
        className="ant-card ant-card-bordered"
      >
        <div
          className="ant-card-body"
          style={Object {}}
        >
          <div
            className="ant-row cap-row"
          >
            <div
              className="ant-col-6 cap-column radio-card-icon"
            >
              <div
                className="icons"
              >
                <a>
                  <div
                    className="div-icon"
                  >
                    <svg
                      height={24}
                      width={24}
                    >
                      <defs>
                        <path
                          d="M4 4h16v12H5.17L4 17.17V4zm0-2c-1.1 0-1.99.9-1.99 2L2 22l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2H4zm2 10h12v2H6v-2zm0-3h12v2H6V9zm0-3h12v2H6V6z"
                          id="message_svg__a"
                        />
                      </defs>
                      <g
                        fill="none"
                        fillRule="evenodd"
                      >
                        <mask
                          fill="#fff"
                          id="message_svg__b"
                        >
                          <use
                            xlinkHref="#message_svg__a"
                          />
                        </mask>
                        <use
                          fill="#FFF"
                          fillRule="nonzero"
                          xlinkHref="#message_svg__a"
                        />
                        <g
                          fill="#091E42"
                          mask="url(#message_svg__b)"
                        >
                          <path
                            d="M0 0h24v24H0z"
                          />
                        </g>
                      </g>
                    </svg>
                  </div>
                </a>
              </div>
            </div>
            <div
              className="ant-col-18 cap-column"
            >
              <div
                className="ant-row cap-row radio-card-header"
              >
                <div
                  className="sc-bdVaJa kuwkmp"
                  type="h4"
                >
                  Outbound
                </div>
              </div>
              <div
                className="ant-row cap-row radio-card-content"
              >
                 
                Sending an outbound message is like blasting the message or broadcasting the message to a pre-defined set of users
              </div>
            </div>
          </div>
        </div>
      </div>
    </span>
  </label>
  <label
    className="ant-radio-button-wrapper"
  >
    <span
      className="ant-radio-button"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-button-input"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value={1}
      />
      <span
        className="ant-radio-button-inner"
      />
    </span>
    <span>
      <svg
        className="radio-card-checked"
        height={24}
        viewBox="0 0 48 48"
        width={24}
      >
        <path
          d="M44 24c0 11.045-8.955 20-20 20S4 35.045 4 24 12.955 4 24 4s20 8.955 20 20z"
          fill="#4caf50"
        />
        <path
          d="M34.602 14.602L21 28.199l-5.602-5.598-2.797 2.797L21 33.801l16.398-16.402-2.796-2.797z"
          fill="#ccff90"
        />
      </svg>
      <div
        className="ant-card ant-card-bordered"
      >
        <div
          className="ant-card-body"
          style={Object {}}
        >
          <div
            className="ant-row cap-row"
          >
            <div />
            <div
              className="ant-col-18 cap-column"
            >
              <div
                className="ant-row cap-row radio-card-header"
              >
                <div
                  className="sc-bdVaJa kuwkmp"
                  type="h4"
                >
                  Dynamic
                </div>
              </div>
              <div
                className="ant-row cap-row radio-card-content"
              >
                 
                for sending messages to users based on their actions in real time
              </div>
            </div>
          </div>
        </div>
      </div>
    </span>
  </label>
  <label
    className="ant-radio-button-wrapper"
  >
    <span
      className="ant-radio-button"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-button-input"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value={2}
      />
      <span
        className="ant-radio-button-inner"
      />
    </span>
    <span>
      <svg
        className="radio-card-checked"
        height={24}
        viewBox="0 0 48 48"
        width={24}
      >
        <path
          d="M44 24c0 11.045-8.955 20-20 20S4 35.045 4 24 12.955 4 24 4s20 8.955 20 20z"
          fill="#4caf50"
        />
        <path
          d="M34.602 14.602L21 28.199l-5.602-5.598-2.797 2.797L21 33.801l16.398-16.402-2.796-2.797z"
          fill="#ccff90"
        />
      </svg>
      <div
        className="ant-card ant-card-bordered"
      >
        <div
          className="ant-card-body"
          style={Object {}}
        >
          <div
            className="ant-row cap-row"
          >
            <div />
            <div
              className="ant-col-18 cap-column"
            >
              <div
                className="ant-row cap-row radio-card-header"
              >
                <div
                  className="sc-bdVaJa kuwkmp"
                  type="h4"
                >
                  Survey
                </div>
              </div>
              <div
                className="ant-row cap-row radio-card-content"
              >
                 
                for taking survey or feedback from the audience
              </div>
            </div>
          </div>
        </div>
      </div>
    </span>
  </label>
  <label
    className="ant-radio-button-wrapper"
  >
    <span
      className="ant-radio-button"
      style={Object {}}
    >
      <input
        checked={false}
        className="ant-radio-button-input"
        disabled={false}
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        type="radio"
        value={3}
      />
      <span
        className="ant-radio-button-inner"
      />
    </span>
    <span>
      <svg
        className="radio-card-checked"
        height={24}
        viewBox="0 0 48 48"
        width={24}
      >
        <path
          d="M44 24c0 11.045-8.955 20-20 20S4 35.045 4 24 12.955 4 24 4s20 8.955 20 20z"
          fill="#4caf50"
        />
        <path
          d="M34.602 14.602L21 28.199l-5.602-5.598-2.797 2.797L21 33.801l16.398-16.402-2.796-2.797z"
          fill="#ccff90"
        />
      </svg>
      <div
        className="ant-card ant-card-bordered"
      >
        <div
          className="ant-card-body"
          style={Object {}}
        >
          <div
            className="ant-row cap-row"
          >
            <div
              className="ant-col-6 cap-column radio-card-icon"
            >
              <div
                className="icons"
              >
                <a>
                  <div
                    className="div-icon"
                  >
                    <svg
                      height={24}
                      width={24}
                    >
                      <defs>
                        <path
                          d="M18.656 11.355c-.254-.62-.537-1.166-.828-1.575-.579-.814-1.274-1.447-2.08-1.9-1.023-.579-2.274-.88-3.743-.88-1.464 0-2.72.302-3.755.883A6.41 6.41 0 0 0 6.171 9.78c-.29.407-.573.953-.827 1.573a12.74 12.74 0 0 0-.245.649c.272.792.646 1.62 1.073 2.217.579.813 1.274 1.446 2.076 1.896 1.037.583 2.29.886 3.75.886 1.458 0 2.713-.303 3.746-.884a6.46 6.46 0 0 0 2.094-1.913c.421-.59.79-1.407 1.063-2.2a13.057 13.057 0 0 0-.245-.648zM21 12c-.136.466-.663 2.146-1.535 3.366a8.453 8.453 0 01-2.737 2.491c-1.347.758-2.94 1.143-4.73 1.143-1.793 0-3.383-.385-4.73-1.143-1.064-.596-1.98-1.431-2.725-2.48-.878-1.226-1.407-2.911-1.543-3.38.136-.465.665-2.148 1.543-3.38A8.41 8.41 0 0 1 7.27 6.14C8.62 5.383 10.212 5 12.005 5c1.8 0 3.39.383 4.727 1.14a8.344 8.344 0 0 1 2.725 2.48C20.335 9.851 20.864 11.534 21 12zm-6 0c0-.8-.312-1.553-.88-2.12A2.98 2.98 0 0 0 12 9c-.8 0-1.553.312-2.12.88A2.98 2.98 0 0 0 9 12c0 .8.312 1.553.88 2.12A2.98 2.98 0 0 0 12 15c.8 0 1.553-.312 2.12-.88A2.98 2.98 0 0 0 15 12z"
                          id="view-dark_svg__a"
                        />
                      </defs>
                      <g
                        fill="none"
                        fillRule="evenodd"
                      >
                        <mask
                          fill="#fff"
                          id="view-dark_svg__b"
                        >
                          <use
                            xlinkHref="#view-dark_svg__a"
                          />
                        </mask>
                        <use
                          fill="#FFF"
                          fillRule="nonzero"
                          xlinkHref="#view-dark_svg__a"
                        />
                        <g
                          fill="#091E42"
                          mask="url(#view-dark_svg__b)"
                        >
                          <path
                            d="M0 0h24v24H0z"
                          />
                        </g>
                      </g>
                    </svg>
                  </div>
                </a>
              </div>
            </div>
            <div
              className="ant-col-18 cap-column"
            >
              <div
                className="ant-row cap-row radio-card-header"
              >
                <div
                  className="sc-bdVaJa kuwkmp"
                  type="h4"
                >
                  Referal
                </div>
              </div>
              <div
                className="ant-row cap-row radio-card-content"
              >
                 
                for sending messages to acquire new users through existing customers
              </div>
            </div>
          </div>
        </div>
      </div>
    </span>
  </label>
</div>
`;
